modules/ROOT/pages/_partials/installing-mobile-services.adoc

```diff
@@ -29,6 +29,12 @@ $ firewall-cmd --permanent --zone dockerc --add-port 443/tcp
 $ firewall-cmd --reload
 ----
 
+WARNING: Firewall issues may arise on Linux if you try to interact with services on a local cluster from an external device. In order to overcome these issues, you can add the ports to your firewall. Depending on the port your service uses, an example of the ports you may want to add to your firewall are
```

Comment:
isn't this already in the text above?

Reply:
That is for adding a port to the DockerRc, this change is to flag a seperate issue with the Linux firewall, and it is only a problem if you are using a localcluster on Linux with an app on an external device. 
Because of that I raised this issue about a troubleshooting section https://github.com/aerogear/mobile-docs/issues/280, as pieces like this would be more suited to a troubleshooting guide then in the docs as a warning.